web/webplatform/webplatform.go

```diff
@@ -27,17 +30,35 @@ func DistFilename() (string, error) {
 		return "", err
 	}
 
-	_, de := kittehs.FindFirst(des, func(de fs.DirEntry) bool { return distRegex.MatchString(de.Name()) })
-	if de == nil {
+	des = kittehs.Filter(des, func(de fs.DirEntry) bool { return distRegex.MatchString(de.Name()) })
```

Comment:
Why do we need to glob? You can know the exact file name from `VERSION`

Reply:
oh wow, right. will try.